app/metrics.yaml

```diff
@@ -5095,7 +5095,7 @@ top_sites:
       A UUID that is unjoinable with other browser metrics. This ID will not be
       shared with AdM, only for internal uses. This ID is shared across all
       contextual services features.
-    lifetime: ping
+    lifetime: application
```

Comment:
Can we also make sure this is also updated in the docs https://docs.google.com/document/d/1aP8gbxnRp0iIqbsqIyuvTeNLZFz92OKJOiv9kQ0cKVc/edit#heading=h.sl9qon5uav6w

Reply:
I updated the doc, including a comment as to why it was changed from "ping" to "application" lifetime.